src/v2/Apps/Artist/Components/MarketStats.tsx

```diff
@@ -0,0 +1,217 @@
+import {
```

Comment:
If this component is only displayed in the auction results tab, then it should live in `Routes/AuctionResults/Components`